src/qt/test/optiontests.cpp

```diff
@@ -29,3 +30,35 @@ void OptionTests::optionTests()
     });
     gArgs.WriteSettingsFile();
 }
+
+void OptionTests::parametersInteraction()
```

Comment:
This name is fine with me, although it implied general parameter interaction testing. 

Absolutely not critical, but while here, as this test is regression specific; maybe we could add a comment similar to the one in the test above?

Reply:
Added a comment.